.mockery.yaml

```diff
@@ -276,7 +276,10 @@ packages:
       LogPollerWrapper:
   github.com/smartcontractkit/chainlink/v2/core/services/relay/evm/headreporter:
     config:
-      dir: core/internal/mocks
```

Comment:
Coupling our mocks together in a central location doesn't have any benefits when they are only used from the local test package.